package.json

```diff
@@ -31,7 +31,10 @@
     "on-finished": "^2.2.0",
     "on-tasks": "git+https://github.com/RackHD/on-tasks.git",
     "pluralize": "~1.1.2",
-    "ws": "^0.8.0"
+    "ws": "^0.8.0",
+    "os-tmpdir": "~1.0.1",
+    "rimraf": "~2.5.0",
+    "tar": "~2.2.1"
```

Comment:
if you use npm to automatically update the package.json, npm will arrange these modules name in alphabetical order
```bash
npm install os-tmpdir --save     #auto insert this module to the dependencies list of package.json
npm install os-tmpdir --save-dev   #auto insert this module t o the devDependencies list of packjson.json
```